src/seedu/addressbook/data/person/Message.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.person;
+
+public class Message {
+    public static final String EXAMPLE = "This is a test message.";
+    public final String fullMessage;
+
+    public Message(String message) {
+        if(message != null && message != "") {
+            this.fullMessage = message.trim();
+        }else{
+            this.fullMessage = null;
+        }
+    }
+
+    public boolean exist(){
+        if(this.fullMessage == null || this.fullMessage == ""){
+            return false;
+        }else{
```

Comment:
"else" here is redundant.
You can either use **guard clause** or **ternary operator**.